src/IECoreHoudini/SOP_SceneCacheSource.cpp

```diff
@@ -634,6 +637,39 @@ bool SOP_SceneCacheSource::convertObject( const IECore::Object *object, const st
 			}
 		}
 		
+		if ( params.tagGroups )
+		{
+			static UT_StringMMPattern convertTagFilter;
+			convertTagFilter.compile("ObjectType:*");
```

Comment:
I think this needs to be protected with an if, otherwise its being re-compiled each time instead of reused.